php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2090,7 +2090,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      * @return array The age of the candidate in an array containing years, months
      *         and days.
      */
-    function getCandidateAge(string $date): array
+    function getCandidateAge(string $date = null): array
```

Comment:
```suggestion
    function getCandidateAge(?string $date = null): array
```

Reply:
You'll also need to change line 2087 to `?string` or `string|null`. Together these changes will fix your phan error.